modules/microshift-config-yaml.adoc

```diff
@@ -6,79 +6,41 @@
 [id="microshift-config-yaml_{context}"]
 = Using a YAML configuration file
 
-Configure your {product-title} using a YAML file. The values specified in your configuration file are used when you execute a command.
-// Q: can these values be overriden with CLI flags?
+{product-title} searches for a configuration file in the user-specific directory, `~/.microshift/config.yaml`, then the system-wide `/etc/microshift/config.yaml` directory. Create the configuring file, specifying any settings that should override the defaults, before starting MicroShift.
```

Comment:
This is a concept module, but the sentence "Create the configuring file..." makes it sound more like a procedure, so this is a rewording suggestion:
```suggestion
{product-title} searches for a configuration file in the user-specific directory, `~/.microshift/config.yaml`, then the system-wide `/etc/microshift/config.yaml` directory. You must create the configuration file and specify any settings that should override the defaults before starting {product-title}.
```